src/controllers/v1/translator.js

```diff
@@ -0,0 +1,32 @@
+import axios from 'axios';
+import dotenv from 'dotenv';
+import Response from '../../utils/response';
+
+dotenv.config();
```

Comment:
You already imported  dotenv in the server.js file. 
you shouldn't need it here

Reply:
I didn't realise that a single import in the root file would do. Thanks.